docs/codeql/support/reusables/frameworks.rst

```diff
@@ -122,19 +122,20 @@ JavaScript and TypeScript built-in support
 Python built-in support
 ====================================
 
-.. csv-table:: 
+.. csv-table::
    :header-rows: 1
    :class: fullWidthTable
    :widths: auto
 
    Name, Category
-   Bottle, Web framework
```

Comment:
What happened to e.g. Bottle?
It seems to still be supported? https://github.com/github/codeql/search?q=Bottle

Reply:
We're in the process of moving our queries away from points-to, and use the same approach as the CodeQL libraries for JS does with type-tracking and API graphs. This means that we need to re-model libraries, and so far Bottle has not been prioritized.